samples/bluetooth/peripheral_uart/src/main.c

```diff
@@ -81,7 +81,7 @@ static const struct bt_data sd[] = {
 	BT_DATA_BYTES(BT_DATA_UUID128_ALL, BT_UUID_NUS_VAL),
 };
 
-#if CONFIG_BT_NUS_UART_ASYNC_ADAPTER
+#if CONFIG_UART_ASYNC_ADAPTER
```

Comment:
```suggestion
#ifdef CONFIG_UART_ASYNC_ADAPTER
```

Reply:
@anangl Out of curiosity. Why  does 'ifdef' suit there better?